src/components/common/TmSessionExtension.vue

```diff
@@ -62,15 +62,6 @@ export default {
       return this.extension.accounts
     }
   },
-  async created() {
-    if (this.$route.params.address && this.$route.params.network) {
-      await this.signIn({
-        address: this.$route.params.address,
-        network: this.$route.params.network
-      })
-      this.$router.push("/portfolio")
-    }
-  },
```

Comment:
Deleting this is not related to these PR changes no? Or it is?

Reply:
Nop, sorry might have slipped in.